### DISCUSSION TITLE
Query context

### DISCUSSION THREAD

**Discussion by briansunter** [briansunter/logseq-plugin-gpt3-openai#191] [Ideas]
Use the results from a query as additional context

**Comment by traverseda** [comment]
I've tried a few of these logseq AI plugins. So far none of them have this feature which I've been finding very frustrating. Most of the time when I'm using AI like this I want it to summarize across queries, "what was my week like", "provide a high level summary of this project", etc.

Is there something that's making this very difficult for logseq plugins?
